CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+We are open to, and grateful for, any contributions made by the community.  By contributing to Effector, you agree to abide by the [code of conduct](https://github.com/zerobias/effector/blob/master/CODE_OF_CONDUCT.md).
+
+## Reporting Issues and Asking Questions
+Before opening an issue, please search the [issue tracker](https://github.com/zerobias/effector/issues) to make sure your issue hasn't already been reported.
+
+## Development
+
+Visit the [Issue tracker](https://github.com/zerobias/effector/issues) to find a list of open issues that need attention.
+
+Fork, then clone the repo:
+```
+git clone https://github.com/your-username/effector.git
```

Comment:
[gitpod](https://gitpod.io/#https://github.com/zerobias/effector) is [configured](https://github.com/zerobias/effector/blob/master/.gitpod.yml) as well